### PR TITLE
Fix bump version

### PR DESCRIPTION
**Pull request by ctmbl**
This PR aims more to be a look into what we maybe miss during the first version bump than a real addition to the project, it personally raised for me many question on how to rightly version a software and in the iscsc.fr case a website.

### CODE REVIEW

**Comment by amtoine** [comment]
> This PR aims more to be a look into what we maybe miss during the first version bump than a real addition to the project, it personally raised for me many question on how to rightly version a software and in the iscsc.fr case a website.
>
i'm not sure what this is about then :thinking:

**maybe you want to make this a draft?**

**Comment by ctmbl** [comment]
You're right it should be a DRAFT :wink: but you still can review the changes I made

File: backend/package.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
-  "name": "server",
+  "name": "iscsc.fr-backend",
   "version": "0.1.0",
   "description": "Backend of the iscsc.fr website",
+  "private": true,
```
Comment (amtoine) [inline backend/package.json:5]: simply not sure what `private` is supposed to do or enforce :yum:

**Review by amtoine — COMMENTED** [review]
looks better :thumbsup:

File: backend/package.json (hunk 1 above)
Comment (ctmbl) [reply backend/package.json:5]: It simply forbid to publish package via `npm publish` https://docs.npmjs.com/cli/v6/configuring-npm/package-json#private

**Comment by atxr** [comment]
Is it ready for review?

File: backend/package.json (hunk 1 above)
Comment (amtoine) [reply backend/package.json:5]: makes sense!

**Comment by amtoine** [comment]
> Is it ready for review?
>
you marked it as such yourself right? :confused:
@ctmbl ?

**Comment by ctmbl** [comment]
@atxr yes it is :wink:

**Comment by ctmbl** [comment]
In particular I would really like any opinion on the [unused fields](https://docs.npmjs.com/cli/v6/configuring-npm/package-json/) of the `package.json` as well as `name` and `description`.

**Comment by atxr** [comment]
I didn't get the point of this PR sorry can you explain it agan?

**Comment by amtoine** [comment]
> In particular I would really like any opinion on the [unused fields](https://docs.npmjs.com/cli/v6/configuring-npm/package-json/) of the `package.json` as well as `name` and `description`.
>
maybe we just don't need them :thinking:

otherwise, you could set default values for each one of them and then we would be able to discuss these values

**Comment by ctmbl** [comment]
@atxr 

> I didn't get the point of this PR sorry can you explain it again?

you're right I've not been clear enough here, in fact this PR has 2 points:
 - first, arise the question about the `package-lock.json`s (we talked about together in private) which hadn't been updated of the 1st version bump (you can check it with `git checkout v0.1.0 && cd backend && npm install && git diff`), **especially** the `./package-lock.json` that can look useless because we don't run `npm install` at root but what if we need to in the future? so here the questions: 
 **Should we run `npm install` when bumping version to update `package-lock.json`s? Should we run it at root? or include it to the `.gitignore`?** 
 - second, simply make the different `package.json` more consistent by adding/updating `name` and `description`

Sure these 2 things aren't of great importance :sweat_smile: but I'm a perfectionist I like things properly done :sweat_smile:

**Comment by ctmbl** [comment]
@amtoine 
> maybe we just don't need them thinking

Absolutely fine to me but I had to ask!

**Comment by atxr** [comment]
Ok it's clearer!
- For the first point, I think we should include the package-lock.json and switch every `npm install` to `npm ci`
However the package-lock.json in the root directory is useless because it will be empty (no packages are installed in root)
- for the second point, adding the name parameter looks good!

**Comment by ctmbl** [comment]
@atxr 
ok so should I remove the root `package-lock.json`?

**Comment by atxr** [comment]
> @atxr 
> ok so should I remove the root `package-lock.json`?

Yes I think so!

**Comment by amtoine** [comment]
feel free to ping me with a new review when needed :+1:

**Comment by ctmbl** [comment]
@atxr @amtoine 
`package-lock.json` has been removed I think we're able to merge this

**Review by amtoine — APPROVED** [review]
looks ok to me

**Review by atxr — APPROVED** [review]
That's better! Let's merge!

**Pull request merged**